components/brave_new_tab_ui/storage.ts

```diff
@@ -10,7 +10,7 @@ import { debounce } from '../common/debounce'
 const keyName = 'new-tab-data'
 
 const defaultState: NewTab.State = {
-  showBackgroundImage: true,
```

Comment:
I believe we wanted to keep the background a picture as to not scare our existing users: https://github.com/brave/brave-browser/issues/4995

Reply:
This does not change the default to false - the default is controlled by the preference object. What this does is have the store _start_ with it set to off, and then when the preferences are retrieved either turns it on or keeps it off.

In practice, there is no delay between first render and retrieving the preferences (which probably happens before render). But if that were to happen, now the items will gracefully display when ready, instead of potentially flashing on at first render and then back off again at first preference retrieval.